packages/fx-core/templates/plugins/resource/frontend/bicep/module.template.bicep

```diff
@@ -0,0 +1,7 @@
+
+module frontendHostingProvision '{{fx-resource-frontend-hosting.modules.frontendHostingProvision.path}}' = {
+  name: 'frontendHostingDeploy'
```

Comment:
I know that our provision step is the deployment for ARM, but this naming is still confusing for me.
You have a module named xxProvision and the module's name is xxDeploy.

Reply:
This should be changed to `frontendHostingProvision`. Thx. Fixed.